example/App.vue

```diff
@@ -24,6 +24,16 @@
     <vue-embed-gist 
       gist-id="8bab656a910829ab9c32d7700c570be6" 
       file="index.js"/>
+    <br>
+    <h3>Change the gist id dynamically</h3>
+    <button 
+      class="button" 
+      @click="changeGistId">
+      <span>Click Me</span>
+    </button>
+    <vue-embed-gist 
+      gist-id="gistIdValue"/>
```

Comment:
```suggestion
      :gist-id="gistIdValue"/>
```
This is why the gists were not loading. Please fix this.

![image](https://user-images.githubusercontent.com/883233/94349173-1dc9f080-fff7-11ea-9f01-bd63a552979c.png)

Reply:
As per feedback, I've done the changes please let me know anything needed